packages/website/src/examples/Table/withAction.example.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import {IActionOptions, TableHOC, TableRowConnected, tableWithActions} from '@coveord/plasma-react';
+import {compose} from 'redux';
+import {loremIpsum} from 'lorem-ipsum';
+
+export default () => (
+    <TableComposed
+        id="tableId"
+        className="table"
+        data={dataForRows}
+        renderBody={(data: any) => generateRows(data, 'tableId')}
+        tableHeader={renderHeader()}
+        showBorderTop
+        showBorderBottom
+    />
+);
+
+const TableComposed = compose<any>(tableWithActions())(TableHOC);
```

Comment:
Do you need the any here?
```suggestion
const TableComposed = compose(tableWithActions())(TableHOC);
```

Reply:
without it I get those errors :confused:  I stole the <any> idea from the pr of Gael for the SingleSelect composed :sweat_smile: 

![image](https://user-images.githubusercontent.com/63734941/160673323-241faa1f-be4e-4874-8eb6-0dbab376cccc.png)

![image](https://user-images.githubusercontent.com/63734941/160673677-420fc5b0-7518-47f8-b230-7866d0fefbd5.png)

I still don't get it :woman_shrugging: